packages/components/src/mobile/utils/alignments.native.js

```diff
@@ -3,12 +3,22 @@ export const WIDE_ALIGNMENTS = {
 		wide: 'wide',
 		full: 'full',
 	},
-	excludeBlocks: [ 'core/columns', 'core/heading' ],
+	innerContainers: [ 'core/group', 'core/columns', 'core/column' ],
+	excludeBlocks: [ 'core/heading' ],
 };
 
 export const ALIGNMENT_BREAKPOINTS = {
 	wide: 1024,
 	large: 820,
 	medium: 768,
 	small: 680,
+	mobile: 480,
 };
+
+export const isFullWidth = ( align ) =>
+	align === WIDE_ALIGNMENTS.alignments.full;
+
+export const isWideWidth = ( align ) =>
+	align === WIDE_ALIGNMENTS.alignments.wide;
+
+export const isWiderThanMobile = ( width ) => width > 480;
```

Comment:
What do you think if we use the value of `mobile` in `ALIGNMENT_BREAKPOINTS`? Just in case it's changed in the future and someone doesn't see this value.

Reply:
Actually I've added `mobile` breakpoint to `ALIGNMENT_BREAKPOINTS`, but missed using it right there